packages/cta-cli/src/cli.ts

```diff
@@ -361,6 +361,7 @@ Remove your node_modules directory and package lock file and re-install.`,
       '--addon-details <addon-id>',
       'show detailed information about a specific add-on',
     )
+    .option('--git', 'create a git repository', false)
     .option('--no-git', 'do not create a git repository')
```

Comment:
Although `--no-git` is now redundant (since --git defaults to false), it is intentionally kept for backward compatibility. Happy to remove it if unnecessary.